components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import { SearchIcon } from "../Icons/Icons";
+import { SearchBarContainer, SearchInput, IconWrapper } from './styles';
+
+
+function SearchBar({ onSearch }) {
+  const [searchTerm, setSearchTerm] = useState("");
+
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    onSearch(searchTerm);
+    setSearchTerm("");
+  };
+  return (
+    <SearchBarContainer onSubmit={handleSubmit}>
+      <SearchInput
+        type="text"
+        value={searchTerm}
+        onChange={(e) => setSearchTerm(e.target.value)}
+        placeholder="Suchen..."
+      />
+      <IconWrapper onClick={handleSubmit}>
```

Comment:
If IconWrapper is a `button`, which i think it is, then you can add type="submit" and remove the onClick handler. 
```suggestion
      <IconWrapper type="submit">
```